tfaot-compile.file

```diff
@@ -0,0 +1,33 @@
+## tfaot common compilation and requirement file
+## specs including this file should provide:
+## 1. a variable %{aot_config}, pointing to the aot config file of the model to compile (required)
+## 2. a variable %{aot_source}, referring to a fetched source to unpack during %prep (optional)
+##    (in this case a "Source" should be defined and %{aot_source} is likely %{n}-%{realversion})
+
+BuildRequires: py3-cms-tfaot
+Requires: tensorflow-xla-runtime
+
+%prep
+%if "%{?aot_source}"
+%setup -n %{aot_source}
+%endif
+
+%build
+cms_tfaot_compile \
+  --aot-config "%{aot_config}" \
+  --tool-name "%{n}" \
+  --tool-base "%{i}" \
+  --output-directory compiled_model
+
+%install
+mkdir -p %{i}/lib
+mv compiled_model/*.o %{i}/lib/
+
+mkdir -p %{i}/include/%{n}
+mv compiled_model/*.h %{i}/include/%{n}
+
+mkdir -p %{i}/etc/scram.d
+mv compiled_model/%{n}.xml %{i}/etc/scram.d/
+
+%post
+%{relocateConfig}etc/scram.d/%{n}.xml
```

Comment:
@riga, there is build path hardcoded in in `tfaot-model-*/version/include/tfaot-model-test-multi/test_multi.h`. Though it is in comment section but it is better to run relocation on these generated header files. I would suggest to add the following line here to relocated all generated header files
```
%relocateConfigAll include/%{n} *.h
```



[a] 
- https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e51cbd/38438/external_checks/relocate/tfaot-model-test-multi.txt
- https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e51cbd/38438/external_checks/relocate/tfaot-model-test-simple.txt

e.g.
```
#ifndef TFAOT_MODEL_TEST_MULTI_H
#define TFAOT_MODEL_TEST_MULTI_H

/*
 * Auto-generated AOT wrapper for
 *   model path  : /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/py3-cms-tfaot/1.0.0-b8bfd2f94522378f26f348963d4ff4ac/share/test_models/multi/saved_model
 *   prefix      : test_multi
 *   namespace   : tfaot_model
 *   class name  : test_multi
 *   batch sizes : 1, 2, 4
 */
```

Reply:
Added :+1:
Plus, the last commit includes some final changes to the aot python tools.